modin/numpy/__init__.py

```diff
@@ -11,27 +11,28 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+import numpy
+from packaging import version
+
 from . import linalg
 from .arr import array
 from .array_creation import ones_like, tri, zeros_like
 from .array_shaping import append, hstack, ravel, shape, split, transpose
-from .constants import (
-    NAN,
-    NINF,
-    NZERO,
-    PINF,
-    PZERO,
-    Inf,
-    Infinity,
-    NaN,
-    e,
-    euler_gamma,
-    inf,
-    infty,
-    nan,
-    newaxis,
-    pi,
-)
+from .constants import e, euler_gamma, inf, nan, newaxis, pi
+
+if version.parse(numpy.__version__) < version.parse("2.0.0b1"):
```

Comment:
```suggestion
if version.parse(numpy.__version__) < version.parse("2.0.0"):
```
Should we do this way (now or maybe later)?

Reply:
I think it's best to leave it like this even when 2.0.0 comes out, since users will still be able to install pre-release versions that definitely won't work if we change this condition.
```python
>>> version.parse("2.0.0b1") < version.parse("2.0.0")
True
```